src/commands/settings/automod.ts

```diff
@@ -1087,34 +1087,34 @@ const callback = async (interaction: CommandInteraction): Promise<void> => {
         if (i.isButton()) {
             await client.database.guilds.write(interaction.guild.id, { filters: current });
             await client.memory.forceUpdate(interaction.guild.id);
-            config = current;
+            config = _.cloneDeep(current);
             current = _.cloneDeep(config);
```

Comment:
Why is an additional deep clone needed here? Given that current is deep cloned before being mutated again it'd seem to me that unless we save a copy of it somewhere there's no need for this

As far as I can tell either this or L1091 is possibly not needed